dss/events/handlers/sync.py

```diff
@@ -0,0 +1,82 @@
+import os, sys, datetime, json, time, hashlib
+from contextlib import closing
+
+import boto3
+import botocore.session
+import urllib3
```

Comment:
Can you clarify what you mean here? It's good practice to order imports into 3 groups: stdlib, third-party, and local library/relative. Within each group, do you want them ordered alphabetically?